discovery-provider/src/api/v1/users.py

```diff
@@ -80,53 +84,59 @@ def get_single_user(user_id, current_user_id):
     return success_response(user)
 
 
-USER_ROUTE = "/<string:user_id>"
+USER_ROUTE = "/<string:id>"
 
 
 @ns.route(USER_ROUTE)
 class User(Resource):
     @record_metrics
     @ns.doc(
         id="""Get User""",
-        params={"user_id": "A User ID"},
+        description="Gets a single user by their encoded user ID",
+        params={"id": "The user's id"},
```

Comment:
In the track's route ns doc id is labeled `params={"track_id": "A Track ID"},` do we want the same wording in the docs for consistency

Reply:
Yes!